### PR TITLE
[6492] implementation jinja2 templates

### PR DESCRIPTION
**Pull request by ADubielSTX**
**Story / Bug id**: https://bugzilla.stxnext.pl/show_bug.cgi?id=6492
**Project / App**: Onboarding - Presence Analyzer
**Reference person**: @ADubielSTX
**Description**: Replacing single HTML files for each view to jinja2 templates.

**Migrations**:

**Rollback**:

**New imports / dependencies**: 

**Unit Tests**:

**What tests do I need to run to validate this change**: unittests and check in web browser that views displays fine.

## New or Changed JS and / or HTML Files:
Deleted old static HTML:
src/presence_analyzer/static/mean_time_weekday.html
src/presence_analyzer/static/presence_start_end.html
src/presence_analyzer/static/presence_weekday.html

Add new template HTML, CSS and JS:
src/presence_analyzer/templates/layout.html
src/presence_analyzer/templates/mean_time_weekday.html
src/presence_analyzer/templates/notFound.html
src/presence_analyzer/templates/presence_start_end.html
src/presence_analyzer/templates/presence_weekday.html
src/presence_analyzer/static/css/styles.css
src/presence_analyzer/static/js/mean_time_weekday.js
src/presence_analyzer/static/js/parse_interval.js
src/presence_analyzer/static/js/presence_start_end.js
src/presence_analyzer/static/js/presence_weekday.js

### CODE REVIEW

**Comment by ADubielSTX** [comment]
Gist with PEP8, pylint, unitttests results:
https://gist.github.com/ADubielSTX/239ae122286b4cfe4a0b

File: src/presence_analyzer/templates/layout.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            {%
+            	set navigation = [
+            		("presence_weekday.html", "Presence by weekday"),
+            		("mean_time_weekday.html", "Presence mean time"),
+            		("presence_start_end.html", "Presence start-end"),
+            	]
+            %}
+            <ul>
+            	{% for menu_url, menu_title in navigation %}
+            		<li
```
Comment (piasekps) [inline src/presence_analyzer/templates/layout.html]: its a bit annoying read splited like that code, my suggestion would be keep a tag in one line even if that will be more than 80 characters.

File: src/presence_analyzer/views.py [hunk 2]
```diff
@@ -92,3 +93,14 @@ def presence_start_end(user_id):
         for weekday, day in enumerate(weekdays)
     ]
     return result
+
+
+@app.route('/<string:temp_name>', methods=['GET'])
+def render_all(temp_name):
```
Comment (piasekps) [inline src/presence_analyzer/views.py]: I don't see any unittests for that method!

**Comment by piasekps** [comment]
Review finished, except missing unittests rest looks very good!

File: src/presence_analyzer/views.py (hunk 2 above)
Comment (ADubielSTX) [inline src/presence_analyzer/views.py]: Your eyes works very well!
I fixed it.

**Pull request merged**